reservacionVuelo/Screens/SignUp.js

```diff
@@ -1,16 +1,111 @@
 import React, { Component } from 'react';
-import { View } from 'react-native';
+import { StyleSheet, Text, TextInput, TouchableHighlight, View } from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 
 export class SignUp extends Component {
     constructor() {
-
+        super()
     }
 
     render() {
         return (
-            <View>
+            <View style={styles.mainContainer} >
+                <Text style={styles.titleStyle}>Sign Up</Text>
+                {/**TextInputs */}
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>First Name</Text>
+                    <TextInput style={styles.textInputStyle}></TextInput>
+                </View>
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>E-mail *</Text>
+                    <TextInput style={styles.textInputStyle}></TextInput>
+                </View>
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>Password *</Text>
+                    <TextInput style={styles.textInputStyle} secureTextEntry={true}></TextInput>
+                </View>
+
+                {/**CheckBoxes */}
+                <View style={styles.checkBoxAlignment}>
+                    <CheckBox tintColors={{ true: '#5974F5' }} />
+                    <Text style={styles.bottomTextStyle}>I agree to the <Text style={styles.hyperlinkText}>Terms</Text> and <Text style={styles.hyperlinkText}>Privacy Policy</Text></Text>
+                </View>
+                <View style={styles.checkBoxAlignment}>
+                    <CheckBox tintColors={{ true: '#5974F5' }} />
+                    <Text style={styles.bottomTextStyle}>Subscribe for select product update</Text>
+                </View>
+
+                {/* Buttons */}
+                <TouchableHighlight style={styles.buttonStyle}>
+                    <Text style={styles.buttonTextStyle}>Sign Up</Text>
+                </TouchableHighlight>
+                <Text style={{ textAlign: 'center', color: '#A3ACB9', fontSize: 15, paddingVertical: 15 }}>or</Text>
```

Comment:
In "_style_", since you are using the "_StyleSheet_",  you can also name a style for this component.  :3